### PR TITLE
Add new commanline for Sample Unified

### PR DESCRIPTION
**Pull request by ddubuc**

### CODE REVIEW

File: Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs [hunk 1]
```diff
@@ -82,189 +69,93 @@ private static void Main(string[] args)
...
-        }
-      }
+      var rootCommand = new RootCommand("Samples for unifiedAPI: Binary for simple tests and quick benchmarks");
+      var pTaskCommand = new Command("ptask",
+                                     "Execute Parallel task with different number of task and/or different size of payload");
+      var numberTaskOption = new Option<int>("--nbTask",
+                                             description: "An option to set the number of task",
+                                             getDefaultValue: () => 100);
+      var numberOfDoubleElement = new Option<int>("--nbElement",
+                                                  description: "An option to set the number of Double element a vector as Client payload",
```
Comment (dbrasseur-aneo) [inline Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs:80]:
```suggestion
                                                  description: "An option to set the number of Double elements in a vector as Client payload",
```

File: Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs [hunk 2]
```diff
@@ -82,189 +69,93 @@ private static void Main(string[] args)
...
-            logger_.LogInformation("Result is " + string.Join(", ",
-                                                              values.ConvertToArray()));
-            break;
-        }
-      }
+      var rootCommand = new RootCommand("Samples for unifiedAPI: Binary for simple tests and quick benchmarks");
+      var pTaskCommand = new Command("ptask",
+                                     "Execute Parallel task with different number of task and/or different size of payload");
+      var numberTaskOption = new Option<int>("--nbTask",
+                                             description: "An option to set the number of task",
```
Comment (dbrasseur-aneo) [inline Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs:77]:
```suggestion
                                             description: "An option to set the number of tasks",
```

File: Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs [hunk 3]
```diff
@@ -82,189 +69,93 @@ private static void Main(string[] args)
...
-            break;
-          case byte[] values:
-            logger_.LogInformation("Result is " + string.Join(", ",
-                                                              values.ConvertToArray()));
-            break;
-        }
-      }
+      var rootCommand = new RootCommand("Samples for unifiedAPI: Binary for simple tests and quick benchmarks");
+      var pTaskCommand = new Command("ptask",
+                                     "Execute Parallel task with different number of task and/or different size of payload");
```
Comment (dbrasseur-aneo) [inline Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs:75]:
```suggestion
                                     "Execute Parallel tasks with different number of tasks and/or different sizes of payload");
```

File: Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs [hunk 4]
```diff
@@ -82,189 +69,93 @@ private static void Main(string[] args)
...
+                                                         workLoadTimeInMs);
+                              },
+                              numberTaskOption,
+                              numberOfDoubleElement,
+                              numberOfBytes,
+                              workLoadTimeInMs);
+
+
+      var simpleTestCommand = new Command("simple",
+                                          "Execute Simple Unified API test with some Execution");
```
Comment (dbrasseur-aneo) [inline Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs:125]: With some execution ???

File: Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs [hunk 5]
```diff
@@ -82,189 +69,93 @@ private static void Main(string[] args)
...
+
+      simpleTestCommand.SetHandler(() =>
+                                   {
+                                     logger_.LogInformation("Running Simple execution test with UnifiedApi");
+                                     new SimpleUnifiedAPI(configuration_,
+                                                          factory).SimpleExecution();
+                                   });
+
+      var nTaskCommand = new Command("ntask",
+                                     "Execute several thousand tasks in a batch of execution");
```
Comment (dbrasseur-aneo) [inline Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs:135]:
```suggestion
                                     "Execute several thousands tasks in a batch of execution");
```

File: Samples/UnifiedAPI/ArmoniK.Samples.Client/appsettings.json [hunk 6]
```diff
@@ -42,7 +42,7 @@
   },
 
   "Grpc": {
-    "EndPoint": "http://ANEO-SB2-8454-wsl.local:5001"
+    "EndPoint": "http://a810dc85181ab4ac682d984366e81554-1922853256.eu-west-3.elb.amazonaws.com:5001"
```
Comment (dbrasseur-aneo) [inline Samples/UnifiedAPI/ArmoniK.Samples.Client/appsettings.json:45]: Not a fan of having this in the default config, but I guess in this case it doesn't matter ?

**Review by dbrasseur-aneo — CHANGES_REQUESTED** [review]

File: Samples/UnifiedAPI/ArmoniK.Samples.Client/appsettings.json (hunk 6 above)
Comment (ddubuc) [reply Samples/UnifiedAPI/ArmoniK.Samples.Client/appsettings.json:45]: We nedd to ignore change on this file in gitignore ?

**Review by dbrasseur-aneo — APPROVED** [review]

**Pull request merged**